defaults/k8s_eks_config.yaml

```diff
@@ -17,7 +17,9 @@ eks_nodegroup_role_arn: 'arn:aws:iam::797456418907:role/helm-test-worker-nodes-N
 eks_vpc_cni_version: 'v1.15.3-eksbuild.1'
 
 scylla_version: '5.2.11'
-scylla_mgmt_agent_version: '3.1.0'
+scylla_mgmt_agent_version: '3.2.5'
```

Comment:
why we have those default set for both EKS/GKE/functional ? why can't those be part of test default ? 
are they used in VMs ?

Reply:
Done